internal-dev/cheminf-ext.owl

```diff
@@ -81,7 +81,16 @@
         <rdfs:isDefinedBy>A class of descriptors derived using CDK, the Chemistry Development Kit.</rdfs:isDefinedBy>
         <rdfs:label xml:lang="en">cdk descriptors</rdfs:label>
     </owl:Class>
-    
+
+
+    <!-- http://purl.enanomapper.org/onto/ENM_1200001 -->
+
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_1200000">
+        <rdfs:subClassOf rdf:resource="http://semanticscience.org/resource/CHEMINF_000025"/>
```

Comment:
What about putting this under http://purl.obolibrary.org/obo/PATO_0050001 ?

Reply:
CHEMINF_000025 is a physical descriptor, and a rate can likely be that. If your terms from from experimental data, then it tries to model a physical reality, not a bit of information. For example, `zeta potential` is used as both the outcome of an experiment and as a descriptor (to model surface charge) in some nanoQSAR models.

Reply:
Yes, I was struggling with this difference myself; however, I believe that the dissolution rate will differ between experiments and is not a bit of information that can be described with one entity (such as melting point). I will make an update for this PR, and send you a new one :)

Reply:
I made a new PR, and added the small note that this entity refers to a experimental value (just to make the distinction more clear): 
![image](https://user-images.githubusercontent.com/26277832/161236079-5d62929c-df7c-457f-a0a3-b524d82d2164.png)

https://github.com/enanomapper/ontologies/pull/314

I will close/cancel this PR, since I made two new ones.